File: solutions/javascript/rna-transcription/1/rna-transcription.js
//
// This is only a SKELETON file for the 'RNA Transcription' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const toRna = (nucleotide) => {
  let output ="";

  for (let element of nucleotide) {
    if(element === 'C')
    output += 'G';
    else if(element ==='G')
    output +='C';
    else if (element === 'T')
    output +='A';
    else if(element === 'A')
    output +='U';
    
  };
  return output;
};

File: solutions/csharp/rna-transcription/1/RnaTranscription.cs
public static class RnaTranscription
{
    public static string ToRna(string nucleotide)
    {
        string output = "";
        foreach (char c in nucleotide)
        {
            if (c == 'C')
                output += 'G';
            else if (c == 'G')
                output += 'C';
            else if (c == 'T')
                output += 'A';
            else if (c == 'A')
                output += 'U';
        }
        return output;
    }
}